definitions/contested/json/CaseEventToFields/CaseEventToFields.json

```diff
@@ -3726,8 +3726,20 @@
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "FinancialRemedyContested",
     "CaseEventID": "FR_amendApplication",
-    "CaseFieldID": "applicantSolicitorFirm",
+    "CaseFieldID": "ApplicantOrganisationPolicy",
```

Comment:
Consented have got display condition but contested not: `"FieldShowCondition": "applicantRepresented=\"Yes\" OR  isAdmin=\"No\"",` - is that correct?

Reply:
as discussed, this is in line with other fields